packages/node/src/exec/default-attenuator.js

```diff
@@ -6,6 +6,7 @@
  */
 
 import { endowmentsToolkit } from 'lavamoat-core'
+import { scuttle } from "lavamoat-core/src/scuttle.js"
```

Comment:
This should be exported from the root e.g.

```js
import { scuttle } from 'lavamoat-core'
```

Ideally this is a subpath export such as `lavamoat-core/scuttle` but adding an `exports` field is often fraught with peril and must be done carefully

Reply:
agreed. should we defer to another pr? since this "changes" core pkg? your call

Reply:
yes, it should be in another PR.  sorry